examples/tensorflow/image-classifier/predictor.py

```diff
@@ -0,0 +1,22 @@
+# WARNING: you are on the master branch, please refer to the examples on the branch that matches your `cortex version`
+
+import requests
+import numpy as np
+from PIL import Image
+from io import BytesIO
+
+labels = requests.get(
```

Comment:
Should this be in `init()`? I'm not sure that it should, just wanted you to consider